jupyter_server/serverapp.py

```diff
@@ -1039,7 +1039,7 @@ def template_file_path(self):
     )
 
     kernel_manager_class = Type(
-        default_value=MappingKernelManager,
+        default_value=AsyncMappingKernelManager,
```

Comment:
Since this is changing the default manager class name, I think we need to add logic that addresses user config. If any traits of `MappingKernelManager` are configurable, some users may have config that will no longer be loaded by this new class, and thus, silently ignore their settings.

I see two ways to address this:
1. If those traits are still valid for this new class, we can write an empty configurable `MappingKernelManager` class that `AsyncMappingKernelManager` inherits and thus loads the traits as a subclass.
2. Throw warnings every time a `MappingKernelManager` trait pops up in the config.

Reply:
Thanks for pointing that out @Zsailer. I think option 1 is the best.